rotkehlchen/externalapis/etherscan.py

```diff
@@ -240,9 +252,11 @@ def _query(
                     },
                 )
                 self.warning_given = True
-        else:
-            query_str += f'&apikey={api_key}'
 
+            api_key = ApiKey(ROTKI_INCLUDED_KEYS[self.chain])
+            log.debug(f'Using default etherscan key for {self.chain}')
```

Comment:
Do we need this? Won';t this spam the logs? We can probably see that the default key is being used as api key is also logged in the logs right?

On the other hand seeing this ins the logs would point us to a user who needs to take action. Not sure what's best.